core/themes/seven/css/style.css

```diff
@@ -22,7 +22,11 @@
 [dir="rtl"] .item-list ul {
   margin: 0.25em 1.5em 0.25em 0;
 }
-.item-list ul li,
+.item-list ol>li {
+  list-style-type: decimal;
+  list-style-image: none;
+}
+.item-list ul>li,
```

Comment:
I think we could avoid adding new more-specific selectors here by eliminating the unnecessary selector in `seven.base.css` instead. I filed a PR against this one at https://github.com/klonos/backdrop/pull/13